client/app/rules/component/RuleList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { MdPushPin } from 'react-icons/md'
+import { useGetHouseQuery } from '@/_hooks/api'
+
+const RuleList = () => {
+  const { data: house } = useGetHouseQuery()
```

Comment:
Hmm, I think it is very preference for architecture, but in this case, just only props drilling, so I think the following is better. Two reasons:
1. Data flow is more clear
2.  RuleList does not have undefined possibility 

So the props is like this

```js
type RuleListProps = {
  rules: {
    id: number
    text: string
  }[]
 } 
```